lib/glimesh_web/templates/user_applications/index.html.eex

```diff
@@ -23,6 +23,9 @@
           <p>
             <%= gettext("Want to build a chat bot, a online tool, or other service that integrates with Glimesh? You're in the right place! Create an Application to get started and checkout our API documentation if you need some guidance.") %>
```

Comment:
```suggestion
            <%= gettext("Want to build a chat bot, a online tool, or other service that integrates with Glimesh? You're in the right place! Create an Application to get started and check out our API documentation if you need some guidance.") %>
```

Reply:
@mja00 I was worried this would mess up the updates you just pushed out, we good?